api-v2/src/student/student.controller.ts

```diff
@@ -0,0 +1,123 @@
+import {
+  Controller,
+  Get,
+  Post,
+  Body,
+  Patch,
+  Param,
+  Delete,
+  ParseUUIDPipe,
+  BadRequestException,
+  UseGuards,
+  Req,
+  Query,
+  ParseBoolPipe,
+  DefaultValuePipe,
+} from '@nestjs/common';
+import { StudentService } from './student.service';
+import { CreateStudentDto } from './dto/create-student.dto';
+import { UpdateStudentDto } from './dto/update-student.dto';
+import { DeleteResult, UpdateResult } from 'typeorm';
+import { Student } from './entities/student.entity';
+import { JwtAuthGuard } from 'src/guards/jwt-auth.guard';
+import { DevRouteGuard } from 'src/guards/dev-route.guard';
+import { AuthenticatedRequest } from 'src/auth/interfaces/authenticated-request';
+
+@Controller('students')
+export class StudentController {
+  constructor(private readonly studentService: StudentService) {}
+
+  @UseGuards(JwtAuthGuard)
+  @Get('/me')
```

Comment:
is this route supposed to be `.../students/me`? if so then the prefix / is unnecessary

Reply:
ahh yes, thanks for catching this!